Src/Base/AMReX_PODVector.H

```diff
@@ -512,6 +512,15 @@ namespace amrex
                                    (Allocator const&)(*this));
         }
 
+        /** Set the same value to every element of the vector
+         *
+         * @param a_value the value to assign
+         */
+        void assign (const & T a_value)
```

Comment:
We can also avoid an overload and call this `setVal(value)` in symmetry with MultiFab, what do you think?

Reply:
In MultiFabs, we further overload the assignment operator for `amrex::Real` and/or `int`, for convenience.

Reply:
I think `assign` is fine. It will still be asymmetric unless we add `assign` to MultiFab, which most versions would not make sense.